test/Base_Tests/src/Data/Set_Spec.enso

```diff
@@ -62,6 +62,12 @@ add_specs suite_builder =
             (s1 == s1) . should_be_true
             (s1 == s3) . should_be_false
 
+        group_builder.specify "should be able to convert to text" <|
+            s1 = Set.from_vector ["1", "2", "3"]
+            s2 = Set.from_vector [1, 2, 3]
+            s1.to_text.should_equal "Set{1, 2, 3}"
+            s2.to_text.should_equal "Set{1, 2, 3}"
```

Comment:
Shall we maybe use `.pretty` to be able to distinguish a set of text values from numbers?

i.e. that would give us
```suggestion
        group_builder.specify "should be able to convert to text" <|
            s1 = Set.from_vector ["1", "2", "3"]
            s2 = Set.from_vector [1, 2, 3]
            s1.to_text.should_equal "Set{'1', '2', '3'}"
            s2.to_text.should_equal "Set{1, 2, 3}"
```

Reply:
Done.